fixtures/ed/ntfs/frequencies.txt

```diff
@@ -0,0 +1,2 @@
+trip_id,start_time,end_time,headway_secs
```

Comment:
As mentionned on IRC : OK to modify both ntfs fixtures (as tests are common) and maybe a refactor for us after to test only a good ntfs v0.6 dataset (no need for 4 or 5 testing), but that's another PR.